Directory.Packages.props

```
@@ -2,11 +2,10 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <AspnetVersion>9.0.5</AspnetVersion>
-    <MicrosoftExtensionsVersion>9.0.5</MicrosoftExtensionsVersion>
-    <MicrosoftExtensionsAIVersion>9.5.0</MicrosoftExtensionsAIVersion>
-    <AspireVersion>9.3.0</AspireVersion>
-    <AspireUnstablePackagesVersion>9.3.0-preview.1.25265.20</AspireUnstablePackagesVersion>
+    <AspnetVersion>9.0.7</AspnetVersion>
+    <MicrosoftExtensionsVersion>9.0.7</MicrosoftExtensionsVersion>
+    <AspireVersion>9.4.0-preview.1.25367.4</AspireVersion>
+    <AspireUnstablePackagesVersion>9.4.0-preview.1.25367.4</AspireUnstablePackagesVersion>
```

```suggestion
    <AspireVersion>9.4.0</AspireVersion>
    <AspireUnstablePackagesVersion>9.4.0-preview.1.25378.8</AspireUnstablePackagesVersion>
```

Updating to final versions.